### PR TITLE
chore: add some more questions and answers to the Accessbility Quiz

### LINKED ISSUE #899: [Feature] - Add more beginner friendly accessibility questions
### What general issue would you like to create?

[developerquiz.org](developerquiz.org) currently has over 1200+ quiz questions. 

We are looking to expand the beginner accessibility questions and we encourage other developers to add their quiz questions to the site.

Example of beginner friendly accessibility questions we are looking for.

- what are landmarks, 
- layout structure, 
- min color contrast, 
- min font-size, 
- semantic HTML, 
- labels, buttons, 

You can find the complete list of accessibility questions below.
https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/src/data/accessibility-quiz.ts

You can add your own questions to the top of that file.
**Please first check to make sure your question doesn't already exist before creating a PR.**

Here is the correct format for the questions. 
The distractors represent the incorrect answers. 

```
 {
    Question: "What is web accessibility?",
    Answer: "the practice of building websites that are usable by everyone",
    Distractor1: "a new JavaScript framework",
    Distractor2: "a sorting algorithm",
    Distractor3: "a type of programming paradigm",
    Explanation:
      "Web accessibility means building websites that are usable by as many people as possible.",
    Link: "https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/",
  },
```
For the `link` field, please make sure to use a freeCodeCamp article, freeCodeCamp YouTube video or [W3C content](https://www.w3.org/WAI/).

If you choose to reference a video, please make sure to include the timestamp for the topic covered.

You can read more about how to create timestamps in this [helpful article](https://www.lifewire.com/link-to-specific-part-of-youtube-video-1616414#). 

There are plenty of freeCodeCamp news articles to pull accessibility questions from:
https://www.freecodecamp.org/news/common-accessibility-errors-and-how-to-fix-them/
https://www.freecodecamp.org/news/web-accessibility-best-practices-a11y-tips/
https://www.freecodecamp.org/news/accessibility-testing-tools-for-ada-compliance/
https://www.freecodecamp.org/news/what-is-website-accessibility-18ce00ec990f/
https://www.freecodecamp.org/news/4-principles-for-getting-started-with-website-accessibility-4e85f75730b/

**This issue will not be assigned to anyone and will remain open for multiple contributors.**
**Please do not close this issue**

Happy contributing!

### PR DESCRIPTION
**Pull request by codingmickey**
## Summary of changes

Adding accessibility questions mentioned in Issue https://github.com/freeCodeCamp/Developer_Quiz_Site/issues/899

## Checklist

- [X] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [X] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [X] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [X] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [X] I have provided a summary of my changes.

 #899

### CODE REVIEW

File: src/data/accessibility-quiz.ts [hunk 1]
```diff
@@ -1032,6 +1032,51 @@ const accessibilityQuiz = [
...
+  },
+  {
+    Question: "What is the primary goal of implementing web accessibility practices?",
+    Answer: "To ensure equal access to web content for all users, regardless of their abilities or disabilities.",
+    Distractor1: "To optimize website performance for specific browsers and devices.",
+    Distractor2: "To increase the aesthetic appeal of the website design.",
+    Distractor3: "To restrict access to certain content based on geographical locations.",
+    Explanation: "Web accessibility practices aim to make online content equally accessible to everyone, promoting inclusivity and usability across different user abilities.",
+    Link: "https://www.freecodecamp.org/news/4-principles-for-getting-started-with-website-accessibility-4e85f75730b/"
+  },
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:1044]: we already merged a similar question so let's remove this one

File: src/data/accessibility-quiz.ts [hunk 2]
```diff
@@ -1032,6 +1032,51 @@ const accessibilityQuiz = [
...
+  {
+    Question: "How does ensuring color contrast contribute to web accessibility?",
+    Answer: "It facilitates better readability and comprehension for users with visual impairments or color deficiencies.",
+    Distractor1: "It encourages creative expression and artistic design in website layouts.",
+    Distractor2: "It enhances the emotional impact of the website content on users.",
+    Distractor3: "It standardizes the color scheme across different web pages for consistency.",
+    Explanation: "Sufficient color contrast in web design enhances readability, ensuring that users with visual impairments or color deficiencies can access and comprehend the content effectively.",
+    Link: "https://www.freecodecamp.org/news/4-principles-for-getting-started-with-website-accessibility-4e85f75730b/"
   }
```
Comment (jdwilkin4) [inline src/data/accessibility-quiz.ts:1080]: it looks like we have a similar question in the quiz already so let's remove this one too

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/accessibility-quiz.ts (hunk 2 above)
Comment (codingmickey) [reply src/data/accessibility-quiz.ts:1080]: Oops sorry about it, I tried to check.. but would've missed it

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Comment by codingmickey** [comment]
> Congrats on your first contribution to our repository. 🎉 We look forward to future contributions. 👍

Thanks a lot!! Will try to be more active..

**Pull request merged**